pkg/authorization/workspace_content_authorizer.go

```diff
@@ -116,6 +116,11 @@ func (a *workspaceContentAuthorizer) Authorize(ctx context.Context, attr authori
 		return DelegateAuthorization("logical cluster admin access", a.delegate).Authorize(ctx, attr)
 	}
 
+	// always let external-logical-cluster-admins through
+	if isUser && sets.NewString(attr.GetUser().GetGroups()...).Has(bootstrap.SystemExternalLogicalClusterAdmin) {
```

Comment:
`sets.NewString(attr.GetUser().GetGroups()...` is now used twice. as this fragment is invoked for every request I suggest to store it in a local variable.

Reply:
Thanks, it's actually used three times, now moved to a `groups` variable :+1: